src/tests/reducers/passwordResetReducers.test.js

```diff
@@ -0,0 +1,111 @@
+import {
+  SEND_RESET_LINK_INITIATED,
+  SEND_RESET_LINK_SUCCESS,
+  SEND_RESET_LINK_ERROR,
+  RESET_PASSWORD_SUCCESS,
+  RESET_PASSWORD_ERROR,
+} from '../../actions/types';
+import userReducer from '../../reducers/userReducer';
+
+describe('passwordResetReducers', () => {
+  let initialState;
+  let errorData;
+  let resetPasswordErrorResponse;
+
+  beforeEach(() => {
+    initialState = {
+      sendLinkSuccess: false,
+      resetPasswordloading: false,
+      loading: false,
+      resetPasswordSuccess: false,
+      sendLinkError: {},
+      resetPasswordError: {},
+    };
+  });
+
+  it('should run initial state', () => {
+    expect(userReducer(initialState, {})).toEqual(initialState);
+  });
+
+  it('should begin the loader when SEND_RESET_LINK_INITIATED is changed to true', () => {
+    const action = {
+      type: SEND_RESET_LINK_INITIATED,
+      payload: true,
+    };
+    const currentState = userReducer(initialState, action);
+    expect(currentState).toEqual({
+      loading: false,
```

Comment:
Everyone seems to be doing this which is making this file ambiguous. How about this:

expect(...).toEqual({
    ...initialState,
    resetPasswordloading: action.payload,
});
@RonKbS . If it works, advise the next victims.